README.md

```diff
@@ -1 +1,66 @@
-# LeaderBoard-
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# LeaderBoard
+
+> API TBD
+
+## Built With
+
+- HTML, CSS and Javascript
+- Git and Github
+
+
+## Live Demo (if available)
+
+https://emmykage.github.io/
+
+
+## Getting Started
+
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+  None
+
+### Setup
+click on the code tab on the repository
+download the Zip folder to your computer 
+open file locally
+
+copy the link from the repository
+open terminal 
+enter "git clone [copied link]
+
+Download and view file
+
+
+
+## Authors
+
+👤 Menanya Morris
+
+- GitHub: [@emmykage](https://github.com/emmykage)
+- Twitter: [@emmykage](https://twitter.com/emmykage)
+- LinkedIn: [Menanya Morris](https://www.linkedin.com/in/morris-menanya-a51985104/)
+
+
+## 🤝 Contributing
+https://github.com/Emmykage/Capstone-Project/issues
+
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly update the issue page to this project, as the one here now is a different one.